### PR TITLE
Disable stack-traces in the speedy performance benchmarks.

### PR DESCRIPTION
**Pull request by nickchapman-da**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-lf/interpreter/perf/src/main/scala/com/daml/lf/explore/LoadDarFunction.scala [hunk 1]
```diff
@@ -25,7 +25,10 @@ object LoadDarFunction extends App {
         case (pkgId, pkgArchive) => Decode.readArchivePayloadAndVersion(pkgId, pkgArchive)._1
       }.toMap
 
-    val compiledPackages: CompiledPackages = PureCompiledPackages(packagesMap).right.get
+    val stacktracing: Compiler.StackTraceMode = Compiler.NoStackTrace
```
Comment (hurryabit) [inline daml-lf/interpreter/perf/src/main/scala/com/daml/lf/explore/LoadDarFunction.scala:28]: This line gives _zero_ indication that it is overwriting a virtual method and not declaring a local binding. Scala really sucks in this regard. 😦

Comment (remyhaemmerle-da) [reply daml-lf/interpreter/perf/src/main/scala/com/daml/lf/explore/LoadDarFunction.scala:28]: Indeed I did not see that was overwriting a virtual method. 
In the future, I will ask you to use @S11001001 convention and mark all overwrite method/field (virtual or not) with the `override` keyword.

Comment (nickchapman-da) [reply daml-lf/interpreter/perf/src/main/scala/com/daml/lf/explore/LoadDarFunction.scala:28]: Is this line overwriting a virtual method? I don't think so. It is just a local binding.
That's the intention anyway.

Comment (S11001001) [reply daml-lf/interpreter/perf/src/main/scala/com/daml/lf/explore/LoadDarFunction.scala:28]: It is indeed just a local binding. 😕

Comment (remyhaemmerle-da) [reply daml-lf/interpreter/perf/src/main/scala/com/daml/lf/explore/LoadDarFunction.scala:28]: My bad.

**Pull request merged**